jax/core.py

```diff
@@ -91,8 +98,14 @@ def subjaxprs(jaxpr: Jaxpr) -> Iterator[Jaxpr]:
 
 
 class TypedJaxpr(object):
+  jaxpr: Jaxpr
+  literals: List['Literal']
```

Comment:
This should be just `List[Any]`, in that it's not a list of `Literal` instances but rather just a list of arrays.